pymtl3/datatypes/PythonBits.py

```diff
@@ -528,4 +531,4 @@ def oct( self ):
 
   def hex( self ):
     str = "{:x}".format(int(self._uint)).zfill(((self._nbits-1)//4)+1)
-    return "0x"+str
+    return "0x"+str
```

Comment:
Can we preserve the new line at the end of the file here? Some linter or style guide checker may complain about this.